src/Chainweb/Miner/Config.hs

```diff
@@ -185,19 +190,20 @@ nodeTestMiners = lens _nodeTestMiners (\m c -> m { _nodeTestMiners = c })
 instance ToJSON NodeMiningConfig where
     toJSON o = object
         [ "enabled" .= _nodeMiningEnabled o
-        , "miner" .= _nodeMiner o ]
+        , "miner" .= _nodeMiner o
+        ]
 
 instance FromJSON (NodeMiningConfig -> NodeMiningConfig) where
     parseJSON = withObject "NodeMiningConfig" $ \o -> id
         <$< nodeMiningEnabled ..: "enabled" % o
         <*< nodeMiner ..: "miner" % o
-        <*< nodeTestMiners ..: "testMiners" % o
```

Comment:
?

Reply:
the `testMiners` setting is used in some tests to adjust the hash power when simulating PoW. It used to be a "hidden" configuration setting. But nowadays it is only used programmatically in tests. So, the PR removes it from the config parser.